src/shared/components/FormComponents/SelectController/SelectController.types.ts

```diff
@@ -10,18 +10,20 @@ export type Option = {
   labelKey: string;
   icon?: JSX.Element;
   disabled?: boolean;
-  tooltip?: string;
+  tooltip?: string | JSX.Element;
   hidden?: boolean;
   groupKey?: string;
+  tooltipPlacement?: 'bottom' | 'right';
 };
 
 export type GetMenuItem = {
   labelKey: string;
   value: string | boolean;
   itemDisabled: boolean;
   icon?: JSX.Element;
-  withoutKey?: boolean;
   hidden?: boolean;
+  tooltip?: string | JSX.Element;
+  tooltipPlacement?: 'bottom' | 'right';
```

Comment:
I wonder if it wouldn't be better to type these in the same way as what the Tooltip component actually expects, eg:

```jsx
  tooltip?: React.ReactNode;
  tooltipPlacement?: TooltipProps['placement'];
```

It doesn't make a difference to our actual use case in this PR, but being accurate like this could save us the possibility of having to debug a type error at some point down the line, if we ever iterate on this and we pass a prop value that we would otherwise expect to work.

This goes for both this and the `Option` type, above.